ui/tests/helpers/kv/kv-general-selectors.js

```diff
@@ -13,6 +13,12 @@ export const SELECTORS = {
   secretTab: (tab) => `[data-test-secrets-tab="${tab}"]`,
   emptyStateTitle: '[data-test-empty-state-title]',
   emptyStateMessage: '[data-test-empty-state-message]',
+  kvRow: '[data-test-kv-row]',
+  maxVersions: '[data-test-input="maxVersions"]',
+  casRequired: '[data-test-input="casRequired"]',
+  automateSecretDeletion: '[data-test-ttl-value="Automate secret deletion"]',
+  inlineAlert: '[data-test-inline-alert]',
+  metadataCancel: '[data-test-kv-metadata-cancel]',
```

Comment:
If these are specific to editing then perhaps they should be in their own file, or we could put all of the page selectors in this file and rename it: https://github.com/hashicorp/vault/blob/b3d172f696463628226d0c287a4142ed51d20ce9/ui/tests/helpers/kv/page/kv-secret-details.js#L0-L1